common/check_not_OK_rules_per_rse

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# Copyright European Organization for Nuclear Research (CERN) 2013
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Martin Barisits, <martin.barisits@cern.ch>, 2014
+# - Eric Vaandering, <ewv@fnal.gov>, 2019
+
+"""
+Probe to check the number of not OK rules per rse.
+"""
+
+import sys
+import traceback
+
+from rucio.core import monitor
+from rucio.db.sqla.session import BASE, get_session
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+if BASE.metadata.schema:
+    schema = BASE.metadata.schema + '.'
+else:
+    schema = ''
+
+if __name__ == "__main__":
+    try:
+        session = get_session()
+        sql = 'SELECT count(*),rse_expression from {schema}RULES where state!=\'O\' group by rse_expression'.format(schema=schema)
```

Comment:
Same comment here about rse_expression not being bounded. 

Also, why not combine this and the total one? You can report more than one thing.